src/pages/SingeVideoPage/SingleVideoPage.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import YouTube from "react-youtube";
+import { Navbar } from "../../components";
+import { useTheme, useVideo } from "../../context";
+import { MdPlaylistPlay, MdWatchLater, MdUnpublished } from "react-icons/md";
+import { FaThumbsUp, FaRegThumbsUp } from "react-icons/fa";
```

Comment:
suggestion:here you are importing diiferent icons from different path so you can create one separate file for icons there you can import any icon from react and from that single path(file) you can import anywhere in your code